core/backend/src/test/changesets/ChangeMerging.test.ts

```diff
@@ -43,6 +43,7 @@ describe("ChangeMerging", () => {
     IModelJsFs.copySync(seedFileName, testFileName);
     const upgradedDb = StandaloneDb.openFile(testFileName, OpenMode.ReadWrite);
     createChangeset(upgradedDb);
+    upgradedDb.performCheckpoint();
```

Comment:
i'm not sure why this is necessary, but probably it should be done in `createChangeset` *before* it copies?